app/models/school_data_importer.rb

```diff
@@ -24,18 +24,47 @@ def schools_data_file
   end
 
   def row_to_school(row)
+    local_authority = LocalAuthority.find_or_initialize_by(code: row.fetch("LA (code)"))
+    local_authority.name = row.fetch("LA (name)")
+    local_authority.save!
+
+    local_authority_district = LocalAuthorityDistrict.find_or_initialize_by(code: row.fetch("DistrictAdministrative (code)"))
+    local_authority_district.name = row.fetch("DistrictAdministrative (name)")
+    local_authority_district.save!
+
+    # Not in public gias extract?
```

Comment:
I think this is a combination of fields?

TrustSchoolFlag   (name) | Trusts (name) | SchoolSponsorFlag (name) | SchoolSponsors (name) | FederationFlag (name) | Federations (name)
-- | -- | -- | -- | -- | --


Would be good to check with Data?

Reply:
It has its own check on the todo list, it can be a separate piece of work.